Tasks/Xcode/xcodebuild2.js

```diff
@@ -26,7 +26,11 @@ processInputs() 													// Process inputs to task and create xcv, xcb, impo
 	.then(function(code) {
 		return xcv.exec();											// Print version of xcodebuild / xctool
 	})			
-	.then(execBuild)												// Run main xcodebuild / xctool task
+	.then(execBuild)                                                // Run main xcodebuild / xctool task
+	.then(function(code) {                                          // publish test results
+		publishTestResults(publishResults, testResultsFiles);
+		return code;
```

Comment:
Bryan: Yes its the return code from ExecBuild
Scott: We usually don't bail out in java tasks when publishing fails (we try to print appropriate error messages). Followed the same here. Also will the .fail at the end of calls not handle it?